discojs/discojs-core/src/weights/aggregation.ts

```diff
@@ -1,8 +1,8 @@
 import { List } from 'immutable'
-import tf from '@tensorflow/tfjs'
+import * as tf from '@tensorflow/tfjs'
```

Comment:
I see you've changed the way tf is imported in a lot of places, why is it needed?

Reply:
it was needed while during the implementation of ESM, but it is not needed per se anymore.

the semantic of `import` changes with ESM, before that, it's a mix of TypeScript automagic importer, based on the type of module imported (CommonJS, AMD, ...), with helpers. if you want, you can read the whole issues around it on the [module interop page](https://www.typescriptlang.org/docs/handbook/modules/appendices/esm-cjs-interop.html).

in theses cases, the default import (`import ... from ...) for tfjs (and many other modules) is quasi-equivalent to the namespace import (`import * as ... from ...`) but that's not always the case. I prefer to be explicit with such modules, to show more clearly how the import will be used. there are also probably some benefits with tree shaking thus reduce build time & size.